files/phoneformat.js/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "phoneformat.js",
+  "repo": "albeebe/phoneformat.js",
+  "files": {
+    "basePath": "dist",
+    "include": ["phone-format.js", "phone-format.min.js", "phone-format.min.js.map"]
```

Comment:
Won't the bot reject the add because of too many commits? (the docs say
it's supposed to be exactly two commits - one for each file)

On Mon, Jun 8, 2015, 14:18 Dmitriy Akulov <notifications@github.com> wrote:

> In files/phoneformat.js/update.json
> <https://github.com/jsdelivr/jsdelivr/pull/5206#discussion_r31904776>:
>
> > @@ -0,0 +1,9 @@
> > +{
> > +  "packageManager": "github",
> > +  "name": "phoneformat.js",
> > +  "repo": "albeebe/phoneformat.js",
> > +  "files": {
> > +    "basePath": "dist",
> > +    "include": ["phone-format.js", "phone-format.min.js", "phone-format.min.js.map"]
>
> Instead of that simply use *.min.*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jsdelivr/jsdelivr/pull/5206/files#r31904776>.
>